CC/Sounder/comms-lib.cc

```diff
@@ -83,12 +82,12 @@ int CommsLib::findLTS(const std::vector<std::complex<double>>& iq, int seqLen)
     return best_peak;
 }
 
-size_t CommsLib::find_pilot_seq(std::vector<std::complex<double>> iq,
-    std::vector<std::complex<double>> pilot, size_t seq_len)
+size_t CommsLib::find_pilot_seq(std::vector<std::complex<float>> iq,
```

Comment:
I would probably pass the vectors by reference.  Depending on the use (write / read) add const as applicable.